src/cmake/config.cmake

```diff
@@ -4,11 +4,59 @@ set(PROJECT_NAME "Wazuh Agent")
 # Project version
 set(VERSION "0.1")
 
+# Default Agent Server Url
+set(DEFAULT_SERVER_URL "https://localhost:27000" CACHE STRING "Default Agent Server Url")
+
+# Default Agent Registration Url
+set(DEFAULT_REGISTRATION_URL "https://localhost:55000" CACHE STRING "Default Agent Registration Url")
+
+# Default Agent retry interval
+set(DEFAULT_RETRY_INTERVAL 30 CACHE STRING "Default Agent retry interval")
```

Comment:
There are many variables in this file that are defined as `STRING`, is it possible to use `INTEGER` for them?

Also, since we are adding the description to each, the comments are redundant, can we delete them?

Reply:
- Implemented the `value_or()` change.
- Got rid of redundant comments.
- As for the STRING variables, the  only types available for [CACHE](https://cmake.org/cmake/help/v3.0/command/set.html) variables are FILEPATH, PATH, STRING, BOOL and INTERNAL. I'm not sure how this reflects on c++ constants, which for some reason where defined as `constexpr auto`, I just followed suit when adding new ones.